includes/notes/class-wc-admin-notes.php

```diff
@@ -131,7 +136,21 @@ public static function schedule_unsnooze_notes() {
 		$next  = $queue->get_next( self::UNSNOOZE_HOOK );
 
 		if ( ! $next ) {
-			$queue->schedule_recurring( time(), HOUR_IN_SECONDS, self::UNSNOOZE_HOOK, array(), WC_Admin_Reports_Sync::QUEUE_GROUP );
+			$queue->schedule_recurring( time(), HOUR_IN_SECONDS, self::UNSNOOZE_HOOK, array(), self::QUEUE_GROUP );
+		}
+	}
+
+	/**
+	 * Clears all queued actions.
+	 */
+	public static function clear_queued_actions() {
+		$store = ActionScheduler::store();
+
+		if ( is_a( $store, 'WC_Admin_ActionScheduler_WPPostStore' ) ) {
+			// If we're using our data store, call our bespoke deletion method.
+			$store->clear_pending_wcadmin_actions();
```

Comment:
This method [seems to clear import actions](https://github.com/woocommerce/woocommerce-admin/blob/master/includes/class-wc-admin-actionscheduler-wppoststore.php#L42), right? Shouldn't we create a different method which only cleared note actions?

Reply:
Ya, I think you're right and we should actually extract the actions out of that method to make this reusable.  Updated in https://github.com/woocommerce/woocommerce-admin/pull/2189/commits/13753b7e47869c3b55ead67b34dad87757d99945